integration-tests/src/main/java/org/apache/druid/testing/utils/MsqTestQueryHelper.java

```diff
@@ -95,7 +95,7 @@ public String getQueryURL(String schemeAndHost)
    */
   public SqlTaskStatus submitMsqTask(String sqlQueryString) throws ExecutionException, InterruptedException
   {
-    return submitMsqTask(new SqlQuery(sqlQueryString, null, false, false, false, ImmutableMap.of(), null));
+    return submitMsqTask(new SqlQuery(sqlQueryString, null, false, false, false, ImmutableMap.of("finalizeAggregations", false), null));
```

Comment:
Long fix would be change it to a sql query builder. 
The short fix would be to have the context param as part of the function argument. 
I donot think its correct to have "finalizeAggregations", false as default